test/e2e/scenario/developer-extension/developerExtension.scenario.ts

```diff
@@ -0,0 +1,28 @@
+describe('developer-extension', () => {
+  it('should switch between tabs', async () => {
+    const panel = new DeveloperExtensionPanel()
+    await panel.open()
+    expect(await panel.getSelectedTab()).toEqual('Events')
+
+    await panel.getTab('Infos').click()
+    expect(await panel.getSelectedTab()).toEqual('Infos')
+  })
+})
+
+class DeveloperExtensionPanel {
+  async open() {
+    await browser.url('chrome://extensions')
+    const extensionId = await $('>>>extensions-item').getAttribute('id')
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏How does this selector work?

Reply:
yes, because the element is inside a shadow DOM, I'll add a comment